src/plugins/intel_cpu/tests/functional/subgraph_tests/src/conv_sum_broadcast.cpp

```diff
@@ -108,6 +108,13 @@ class ConcatConvSumInPlaceTest : public testing::WithParamInterface<convSumBroad
 
         auto sum = addSum(conv, inputParams);
 
+        // The case with several convolution consumers is covered only if additional fusings are absent
+        // to prevent fusing check failures because of Reshape
+        if (fusingParams == emptyFusingSpec) {
+            auto shapeOf = std::make_shared<ngraph::opset3::ShapeOf>(sum);
+            sum = std::make_shared<ngraph::opset3::Reshape>(sum, shapeOf, true);
+        }
```

Comment:
You might have noticed that this class might be reused to make some specific implementations. This code has two significant drawbacks: we do not test empty fusing list with only one output and we can not easily disable such tests (with two outputs) if we want to. Could you please add yet another derived test class that would test specially this case - multiple outputs? It clearly can have less parameter combinations in its instance than this one, since it aimed on testing multiple outputs only.

Reply:
Done, thanks